src/Sulu/Bundle/AdminBundle/Serializer/Handler/RepresentationHandler.php

```diff
@@ -11,54 +11,47 @@
 
 namespace Sulu\Bundle\AdminBundle\Serializer\Handler;
 
-use JMS\Serializer\Context;
-use JMS\Serializer\GraphNavigator;
-use JMS\Serializer\Handler\SubscribingHandlerInterface;
-use JMS\Serializer\JsonSerializationVisitor;
-use Sulu\Component\Rest\ListBuilder\CollectionRepresentation;
-use Sulu\Component\Rest\ListBuilder\PaginatedRepresentation;
+use JMS\Serializer\EventDispatcher\Events;
+use JMS\Serializer\EventDispatcher\EventSubscriberInterface;
+use JMS\Serializer\EventDispatcher\ObjectEvent;
+use JMS\Serializer\Metadata\StaticPropertyMetadata;
 use Sulu\Component\Rest\ListBuilder\RepresentationInterface;
-use Sulu\Component\SmartContent\Rest\ItemCollectionRepresentation;
 
 /**
  * @internal
  *
  * This handler workaround some problems with serialize Representation in specific groups
  */
-class RepresentationHandler implements SubscribingHandlerInterface
+class RepresentationHandler implements EventSubscriberInterface
```

Comment:
Shouldn't that class be renamed to `Subscriber` then?

Reply:
Done.